src/utils/nav.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file nav.js
+ *
+ * @brief navigation functions implementations
+ *
+ * @date 09/2021
+ */
+
+import {Navigation} from 'react-native-navigation';
+
+export const GoToPage = (componentId, LandingPage) => {
```

Comment:
```suggestion
export const GoToPage = (componentId, landingPage) => {
```
Posso ter entendido errado, mas aqui deveria ser pascal case, né? (é uma variável, e não uma página)

Reply:
Na verdade devia ser só `page` né hahaha